app/javascript/components/artists/AllArtists.jsx

```diff
@@ -81,14 +98,30 @@ class AllArtists extends React.Component {
           <button onClick={this.getFilteredArtists} className="button-primary bg-indigo w-100"> Apply </button>
         </div>
         <div className="fl w-80 pb5">
-          <h1>Artists</h1>
-          <div className="col-list-3">
-            {artists.filter(artist => artist.works.length > 0).map((artist, i) => {
-              return <ArtistColumnPanel key={i} artist={artist} />
-            })}
+          <div className="flex justify-between items-baseline">
+            <h1>Artists</h1>
+            <nav className="li-indigo pagination" role="navigation" aria-label="Pagination Navigation">
+              <ReactPaginate
+              previousLabel={"\u00ab"}
+              nextLabel="&raquo;"
+              breakLabel={'...'}
+              breakClassName={'break-me'}
+              pageCount={this.state.pageCount}
+              marginPagesDisplayed={2}
+              pageRangeDisplayed={5}
```

Comment:
Hmm this probably shouldn't be hardcoded, it's displaying empty pages bc we don't have a ton of seeded artists

Reply:
Hmm the displaying of empty pages in AllArtists page should not be happening + isn't happening when I'm loading it locally. **marginPagesDisplayed=2** should make it such that if there are more than 4 pages, the first 2 and last 2 pages will be displayed [1,2, 19 and 20 in the pic], and for **pageRangeDisplayed=5** refer to the middle 5 pages [5, 6, 7, 8, 9 in the pic], if there are enough pages. 

Think it should be hardcoded based on we want the pagination bar to look, or if it should be a dynamic value then I can implement that as well!

![image](https://user-images.githubusercontent.com/35573990/54639132-e1085b00-4a49-11e9-8e80-a3ffa97ff482.png)

Reply:
<img width="535" alt="Screen Shot 2019-03-19 at 2 02 03 PM" src="https://user-images.githubusercontent.com/13279205/54641509-9984cd80-4a4f-11e9-9543-b591d04fa1c4.png">

Not sure why I'm seeing so many pages ):

Reply:
Looks like we'll want to use `artists.filter(artist => artist.works.length > 0)` to determine the          `pageCount`